services/apis/fence/fenceSequences.js

```diff
@@ -35,9 +36,22 @@ module.exports = {
    * @returns {Promise<Gen3Response>}
    */
   async linkGoogleAcctMocked(userAcct, expires_in = null) {
-    const linkRes = await fenceTasks.linkGoogleAcctMocked(userAcct, expires_in);
-    fenceQuestions.mockedLinkSuccess(linkRes);
-    return linkRes;
+    const nAttempts = 3;
+    for (let i = 0; i < nAttempts; i += 1) {
```

Comment:
the testing framework has a generic `.retry()` you can append to a given test, this is fine for more granular retries, but we can just retry the entire test too if we're not already

Reply:
I see you've commented in another line that contains the `.retry()` function.
That's right, this is just a granular one because, during the initial experiments with this PR, this very first test was failing most often so another objective I have with these changes is to *fail fast*, hence, the retries on this first check.